doc/api/buffer.md

```diff
@@ -1837,11 +1837,14 @@ console.log(buf.toString('ascii', 0, 5));
 
 const buf2 = Buffer.from('tést');
 
-// Prints: tés
-console.log(buf.toString('utf8', 0, 3));
+// Prints: 74c3a97374
+console.log(buf2.toString('hex'));
```

Comment:
I know this isn’t mentioned in https://github.com/nodejs/node/issues/8971, but the `console.log(buf.toString(…))` statements should probably refer to `buf1`, not `buf`, so it would make sense to fix that, too?